run.py

```diff
@@ -0,0 +1,29 @@
+import cv2
+from ball_detection import detectBall
+from patterntest import determinePattern
+
+# camera = cv2.VideoCapture(0)
```

Comment:
Take a look at argparse for handling command line arguments. That way we can choose to use video or image depending on a command line argument instead of having to comment and uncomment code all the time.